docs/source/load-balancing/default-policy.md

```diff
@@ -10,8 +10,7 @@ for queries with non-local consistency mode is also supported.
 `builder()` method of `DefaultPolicy` returns a new instance of
 `DefaultPolicyBuilder` with the following default values:
 
-- `preferred_datacenter`: `None`
-- `preferred_rack`: `None`
+- `preferences`: `ReplicaLocationPreference::Any`
 - `is_token_aware`: `true`
 - `permit_dc_failover`: `false`
 - `latency_awareness`: `None`
```

Comment:
It's not clear at a first glance what "`preferences`: `ReplicaLocationPreference::Any`" means. I think we should change this section to explain the default parameters in plain English and not refer to the names of internal fields of the load balancing policy.